tx-pool/src/pool.rs

```diff
@@ -456,7 +410,7 @@ impl TxPool {
 
     pub(crate) fn drain_all_transactions(&mut self) -> Vec<TransactionView> {
         let mut txs = CommitTxsScanner::new(&self.pool_map)
-            .txs_to_commit(self.total_tx_size, self.total_tx_cycles)
+            .txs_to_commit(self.pool_map.total_tx_size, self.pool_map.total_tx_cycles)
```

Comment:
Changing it to `usize::max` better reflects that the intent of this fn is to drain all transactions from the pool.

```suggestion
            .txs_to_commit(usize::MAX, usize::MAX)
```